content/en/security/application_security/getting_started/serverless.md

```diff
@@ -0,0 +1,147 @@
+---
+title: AWS Lambda Getting Started with ASM
+kind: documentation
+is_beta: true
+code_lang: serverless
+type: multi-code-lang
+code_lang_weight: 90
+further_reading:
+    - link: "/security/application_security/how-appsec-works/"
+      tag: "Documentation"
+      text: "How Application Security Works"
+    - link: "/security/default_rules/#cat-application-security"
+      tag: "Documentation"
+      text: "OOTB Application Security Management Rules"
+    - link: "/security/application_security/troubleshooting"
+      tag: "Documentation"
+      text: "Troubleshooting Application Security Management"
+    - link: "/security/application_security/threats/"
+      tag: "Documentation"
+      text: "Application Threat Monitoring and Protection"
+    - link: "/security/application_security/risk_management/"
+      tag: "Documentation"
+      text: "Application Risk Management"
+---
+
+{{< callout url="#" header="ASM support for AWS Lambda is in beta" btn_hidden="true" >}}
+  Threat detection is done by using the Lambda extension.
+{{< /callout >}}
+
+You can monitor your functions running in AWS Lambda with Datadog Application Security Management (ASM). 
+
+In general, setting up ASM for AWS Lambda involves:
+
+1. Identifying functions that are vulnerable or are under attack, which would most benefit from ASM. Find them on [the Security tab of your Service Catalog][1].
```

Comment:
The link should point to the Security tab in the Service Catalog - https://app.datadoghq.com/services?env=prod&hostGroup=%2A&lens=Security